libs/ast_generic/AST_generic.ml

```diff
@@ -1332,7 +1332,7 @@ and other_stmt_operator =
   | OS_ThrowFrom
   | OS_ThrowNothing
   | OS_ThrowArgsLocation
-    (* Python2: `raise expr, expr` and `raise expr, expr, exr` *)
+  (* Python2: `raise expr, expr` and `raise expr, expr, exr` *)
```

Comment:
I'm guessing this comment isn't meant to be there?

Reply:
I think its miss aligned; should be above `OS_ThrowFrom`?

Reply:
Wait nvm 😭 I think I mistakingly added it